source/troubleshooting.txt

```diff
@@ -173,3 +173,144 @@ HOMEDRIVE, HOMEPATH, or USERPROFILE are blank
 
 This error might appear if the {+atlas-cli+} can't access your home
 directory.
+
+
+# Are there any known errors to add for ``atlas deployments``?
+
+.. _local-deploy-errors:
+
+Local Deployment Errors
+-----------------------
+
```

Comment:
Can we move this section up to the top of this page? It'll become the most popular and frequently looked for troubleshooting.